pkg/fixedpoint/convert.go

```diff
@@ -590,3 +590,23 @@ func Abs(a Value) Value {
 	}
 	return a
 }
+
+func Clamp(x, min, max Value) Value {
+	if x < min {
+		return min
+	}
+	if x > max {
+		return max
+	}
+	return x
+}
+
+func (x Value) Clamp(min, max Value) Value {
```

Comment:
Please run with -tag dnum mode to verify this.

we have two implementations

Reply:
ok